.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,22 @@
+---
+name: "🐞 Error Report"
+about: 에러를 제보하고 재현 방법을 설명합니다.
+title: "[Error Report] "
+labels: errors
+assignees: ''
+---
+
+## 🔎 어떤 버그인가요?
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)